cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -92,8 +92,12 @@
    * @throws FieldException provided field is not valid
    */
   public CQL2PgJSON(String field) throws FieldException {
-    doInit(field, null);
-    initDbTable();
+    try {
+      doInit(field, null);
+      initDbTable();
+    } catch (NullPointerException e) {
+      logger.log(Level.SEVERE, e.getMessage(), e);
+    }
```

Comment:
Please add a comment that explains why this exception is logged without failing the constructor method.
Can you add a unit test for it?

Reply:
I'll just remove this again. An exception was thrown when classes was not found and it was not logged in whatever parent was calling this. That's why it was added.